src/Website/Views/ContactUs/ContactUs.cshtml

```diff
@@ -0,0 +1,24 @@
+﻿
+@{
+    ViewBag.Title = "Contact Us";
```

Comment:
I think there might be a couple other files that use the ViewBag for the title, but I'd like to eliminate the use of the ViewBag wherever possible, so I actually added a "Title" property to the page base class in a prior commit.

Lets use:
'this.PageTitle = "Contact Us";'
instead.

Also, we should localize that string.